index.html

```diff
@@ -8,6 +8,8 @@
     <link rel="icon" type="image/png" sizes="16x16" href="favicon-16x16.png">
     <link rel="stylesheet" href="./main.css">
     <meta name="theme-color" content="#ffffff">
+    <meta name="apple-mobile-web-app-capable" content="yes">
```

Comment:
There is an edge cache from cloudflare .. so from deploy to live takes around 5 min